litestar/dto/_backend.py

```diff
@@ -750,14 +750,17 @@ def _create_struct_field_meta_for_field_definition(field_definition: TransferDTO
         return None
 
     return msgspec.Meta(
-        gt=kwarg_definition.gt,
+        description=kwarg_definition.description,
+        examples=[e.value for e in kwarg_definition.examples or []],
         ge=kwarg_definition.ge,
-        lt=kwarg_definition.lt,
+        gt=kwarg_definition.gt,
         le=kwarg_definition.le,
-        multiple_of=kwarg_definition.multiple_of,
-        min_length=kwarg_definition.min_length if not field_definition.is_partial else None,
+        lt=kwarg_definition.lt,
         max_length=kwarg_definition.max_length if not field_definition.is_partial else None,
+        min_length=kwarg_definition.min_length if not field_definition.is_partial else None,
```

Comment:
Why is there a need for `is_partial` check and why is it only needed for `min_length` and `max_length`? I didn't quite understand that.

Reply:
This isn't new to this PR - I just sorted them alphabetically so it's showing in the diff.

As to why - I'm not sure either. Maybe something to do with how partial makes the type a union with msgspec's `UnsetType`, but I'm only guessing.

Reply:
Aah got it. I'll try to look into that separately from this PR.